RecoPixelVertexing/PixelTrackFitting/interface/RiemannFit.h

```diff
@@ -33,67 +33,72 @@ using RowVector2Nd = Eigen::Matrix<double, 1, Dynamic, 1, 1, 2 * max_nop>;
 using Matrix5d = Eigen::Matrix<double, 5, 5>;
 using Matrix6d = Eigen::Matrix<double, 6, 6>;
 using Vector5d = Eigen::Matrix<double, 5, 1>;
-using u_int    = unsigned int;
+using u_int = unsigned int;
 
-struct circle_fit {
-  Vector3d par;  //!< parameter: (X0,Y0,R)
-  Matrix3d cov;
-  /*!< covariance matrix: \n
+struct circle_fit
+{
+    Vector3d par;  //!< parameter: (X0,Y0,R)
+    Matrix3d cov;
+    /*!< covariance matrix: \n
       |cov(X0,X0)|cov(Y0,X0)|cov( R,X0)| \n
       |cov(X0,Y0)|cov(Y0,Y0)|cov( R,Y0)| \n
       |cov(X0, R)|cov(Y0, R)|cov( R, R)|
   */
-  int64_t q;  //!< particle charge
-  double chi2;
+    int64_t q;  //!< particle charge
+    double chi2;
 };
 
-struct line_fit {
-  Vector2d par;  //!<(cotan(theta),Zip)
-  Matrix2d cov;
-  /*!<
+struct line_fit
+{
+    Vector2d par;  //!<(cotan(theta),Zip)
+    Matrix2d cov;
+    /*!<
       |cov(c_t,c_t)|cov(Zip,c_t)| \n
       |cov(c_t,Zip)|cov(Zip,Zip)|
   */
-  double chi2;
+    double chi2;
 };
 
-struct helix_fit {
-  Vector5d par;  //!<(phi,Tip,pt,cotan(theta)),Zip)
-  Matrix5d cov;
-  /*!< ()->cov() \n
+struct helix_fit
+{
+    Vector5d par;  //!<(phi,Tip,pt,cotan(theta)),Zip)
+    Matrix5d cov;
+    /*!< ()->cov() \n
       |(phi,phi)|(Tip,phi)|(p_t,phi)|(c_t,phi)|(Zip,phi)| \n
       |(phi,Tip)|(Tip,Tip)|(p_t,Tip)|(c_t,Tip)|(Zip,Tip)| \n
       |(phi,p_t)|(Tip,p_t)|(p_t,p_t)|(c_t,p_t)|(Zip,p_t)| \n
       |(phi,c_t)|(Tip,c_t)|(p_t,c_t)|(c_t,c_t)|(Zip,c_t)| \n
       |(phi,Zip)|(Tip,Zip)|(p_t,Zip)|(c_t,Zip)|(Zip,Zip)|
   */
-  double chi2_circle;
-  double chi2_line;
-  Vector4d fast_fit;
-  int64_t q;  //!< particle charge
-  //  VectorXd time;  // TO FIX just for profiling
-} __attribute__ ((aligned(16)) );
-
-
-template<class C>
-__host__ __device__
-void printIt(C * m, const char * prefix = "") {
+    double chi2_circle;
+    double chi2_line;
+    Vector4d fast_fit;
+    int64_t q;  //!< particle charge
+                //  VectorXd time;  // TO FIX just for profiling
+} __attribute__((aligned(16)));
+
+template <class C>
+__host__ __device__ void printIt(C* m, const char* prefix = "")
+{
 #if RFIT_DEBUG
-  for (u_int r = 0; r < m->rows(); ++r) {
-    for (u_int c = 0; c < m->cols(); ++c) {
-      printf("%s Matrix(%d,%d) = %g\n", prefix, r, c, (*m)(r,c));
+    for (u_int r = 0; r < m->rows(); ++r)
+    {
+        for (u_int c = 0; c < m->cols(); ++c)
+        {
+            printf("%s Matrix(%d,%d) = %g\n", prefix, r, c, (*m)(r, c));
+        }
     }
-  }
 #endif
 }
 
-
 /*!
     \brief raise to square.
 */
 template <typename T>
-__host__ __device__
-inline T sqr(const T a) { return a * a; }
```

Comment:
on the other hand I agree the same comment could be made for the `inline` attribute, for the return type, etc.